libraries/api-extractor/config/post-copy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "copyTo": {
+    "./samples/api-documenter": [ 
+      "../api-documenter/gulpfile.js",
+      "../api-documenter/package.json",
+      "../api-documenter/run.cmd",
+      "../api-documenter/tsconfig.json",
+    ],
+    "./samples/api-documenter/src": [ "../api-documenter/src/**" ]
```

Comment:
Removed, per our discussion.

---
In reply to: [134042661](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134042661) [](ancestors = 134042661,134040834)